lib/MCToIRLifter.cpp

```diff
@@ -123,13 +123,42 @@ void MCToIRLifter::VisitIndirectJump(const remill::Instruction &inst,
   remill::AddTerminatingTailCall(block, intrinsics.jump);
 }
 
+void MCToIRLifter::VisitConditionalIndirectJump(const remill::Instruction &inst,
+                         remill::Instruction *delayed_inst,
+                         llvm::BasicBlock *block) {
+  const auto lifted_func = block->getParent();
+  const auto cond = remill::LoadBranchTaken(block);
+  const auto taken_block = llvm::BasicBlock::Create(ctx, "", lifted_func);
+  remill::AddTerminatingTailCall(taken_block, intrinsics.jump);
```

Comment:
I think this `AddTerminatingTailCall` probably should happen after doing the `VisitDelayedInstruction(inst, delayed_inst, taken_block, true);`